### PR TITLE
Various test fixes

### PR DESCRIPTION
**Pull request by martinpitt**
Broken out from #16942.

### CODE REVIEW

File: test/verify/check-networkmanager-basic [hunk 1]
```diff
@@ -42,6 +42,11 @@ class TestNetworkingBasic(NetworkCase):
         b.wait_in_text("#networking-graphs", "Transmitting")
         b.wait_in_text("#networking-graphs", "Receiving")
 
+        # screenshot assumes non-running firewalld and absent virbr0 (in particular, *3* interfaces)
```
Comment (martinpitt) [inline test/verify/check-networkmanager-basic:45]: [example failure](https://logs.cockpit-project.org/logs/pull-16942-20220208-132850-836165a4-fedora-35/pixeldiff.html#TestNetworkingBasic-testBasic-network-main)

File: test/verify/check-metrics [hunk 2]
```diff
@@ -565,7 +565,8 @@ class TestHistoryMetrics(MachineCase):
             m.execute("! curl --silent --show-error --max-time 10 'http://localhost:44322/series/labels?names=hostname' 2>&1")
 
         # start in a defined state; all test images have pcp and redis pre-installed
-        m.execute(f"systemctl disable --now pmproxy {redis}")
+        m.execute(f"systemctl disable --now pmlogger pmie pmproxy {redis}")
```
Comment (martinpitt) [inline test/verify/check-metrics:568]: [example failure](https://logs.cockpit-project.org/logs/pull-16942-20220208-132850-836165a4-debian-stable/log.html#61-2)

File: test/verify/check-system-journal [hunk 3]
```diff
@@ -607,6 +607,7 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
         self.allow_journal_messages('.*Remote peer disconnected')
         self.allow_browser_errors('.*Failed to get workflows for problem .*: disconnected')
 
+        self.addCleanup(m.execute, "pkill -f '[m]ock-.*-server' || true")
```
Comment (martinpitt) [inline test/verify/check-system-journal:610]: [example failure](https://logs.cockpit-project.org/logs/pull-16942-20220208-132850-836165a4-fedora-35/log.html#46-2)

File: test/verify/check-networkmanager-basic (hunk 1 above)
Comment (martinpitt) [reply test/verify/check-networkmanager-basic]: This fix needs more work. [firewalld isn't a thing on coreos](https://logs.cockpit-project.org/logs/pull-16952-20220208-175753-3f7e58f7-fedora-coreos/log.html#283-2), [follow-up tests get confused](https://logs-https-frontdoor.apps.ocp.ci.centos.org/logs/pull-16952-20220208-175753-3f7e58f7-fedora-35/log.html), and it's [unstable](https://logs.cockpit-project.org/logs/pull-16952-20220208-175753-3f7e58f7-fedora-34/log.html#283). I'll take it out of this series.

**Comment by martinpitt** [comment]
But I added a fix for [another flake](https://logs.cockpit-project.org/logs/pull-16942-20220208-175811-e842353e-fedora-34/log.html#271)

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**